app/services/govpay_refund_details_service.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class GovpayRefundDetailsService < WasteCarriersEngine::BaseService
+  include WasteCarriersEngine::CanSendGovpayRequest
+
+  def run(refund_id:)
+    @refund = GovpayFindPaymentService.run(payment_id: refund_id)
+    raise ArgumentError, "Invalid refund id #{refund_id}" if refund.nil?
+
+    payment_id = refund.refunded_payment_govpay_id
+    # we don't use the payment value but this will force a suitable exception if not found
+    @payment = GovpayFindPaymentService.run(payment_id:)
```

Comment:
Is this argument right? Not sure I understand this syntax, what exactly is the argument?

Reply:
It's the equivalent of `(payment_id: payment_id)`. Rubocop prefers the abbreviated syntax.